docs/openapi/parameters/header/contentType.yml

```diff
@@ -1,4 +1,4 @@
-name: vc-format
+name: content-type
```

Comment:
change to the accept header.... use `application/vc+ld+jwt` and `application/vc+ld+json` for the enum